packages/yoga/src/Dropdown/web/Dropdown.jsx

```diff
@@ -0,0 +1,319 @@
+    top: 0;
+    left: 0;
+
+    width: 100%;
+    height: 100%;
+    padding-right: ${dropdown.button.paddingRight}px;
+
+    border: none;
+    outline: none;
+    background-color: transparent;
+    cursor: ${disabled ? 'not-allowed' : 'pointer'};
+  `}
+`;
+
+const ArrowIcon = styled(Arrow)`
+  ${({
+    isOpen,
+    disabled,
+    selected,
+    theme: {
+      yoga: {
+        colors: { primary },
+        components: { dropdown },
+      },
+    },
+  }) => `
+    fill: ${dropdown.disabled.arrow.fill};
+    ${disabled ? `fill: ${dropdown.disabled.arrow.fill};` : ''};
+    ${selected ? `fill: ${primary[3]};` : ''};
+    transform: rotate(${isOpen ? '180deg' : '0'});
+  `}
+`;
+
+const OptionsList = styled.ul`
+  ${({
+    selected,
+    theme: {
+      yoga: {
+        components: { dropdown },
+      },
+    },
+  }) => `
+    position: absolute;
+    top: 46px;
+    z-index: 1;
+
+    width: 100%;
+    max-height: 140px;
+    margin: 0;
+    padding: 0;
+    overflow-x: scroll;
+    list-style-type: none;
+
+    background: ${dropdown.optionsList.backgroundColor};
+    border: 1px solid ${dropdown.optionsList.border.color};
+    border-top: none;
+
+    ${
+      selected
+        ? `border-color: ${dropdown.selected.optionsList.border.color};`
+        : ''
+    };
+
+    border-radius: 
+      ${dropdown.optionsList.border.radius.top}px 
+      ${dropdown.optionsList.border.radius.right}px 
+      ${dropdown.optionsList.border.radius.bottom}px 
+      ${dropdown.optionsList.border.radius.left}px
+  `}
+`;
+
+const Option = styled.li`
+  ${({
+    isSelected,
+    theme: {
+      yoga: {
+        baseFont,
+        components: { dropdown },
+      },
+    },
+  }) => `
+    display: flex;
+    align-items: center;
+    height: ${dropdown.option.height}px;
+    padding: 
+      ${dropdown.option.padding.top}px 
+      ${dropdown.option.padding.right}px 
+      ${dropdown.option.padding.bottom}px 
+      ${dropdown.option.padding.left}px;
+    cursor: pointer;
+  
+    font-family: ${baseFont.family};
+    font-size: ${dropdown.option.font.size}px;
+    line-height: ${dropdown.option.font.lineHeight}px;
+    
+    font-weight: ${
+      isSelected
+        ? `${dropdown.selected.option.font.weight}`
+        : `${dropdown.option.font.weight}`
+    }; 
+
+    color: ${
+      isSelected
+        ? `${dropdown.selected.option.font.color}`
+        : `${dropdown.option.font.color}`
+    };
+
+    &:hover {
+      background-color: ${dropdown.hover.option.backgroundColor}; 
+    }
+
+    &:last-child {
+      border-radius: 
+        ${dropdown.option.border.radius.top}px 
+        ${dropdown.option.border.radius.right}px 
+        ${dropdown.option.border.radius.bottom}px 
+        ${dropdown.option.border.radius.left}px
+    }
+  `}
+`;
+
+const getSelectedOption = options => {
+  const filteredOptions = options.filter(item => item.selected === true);
+  return filteredOptions.length > 0 ? filteredOptions[0] : null;
+};
```

Comment:
```suggestion
const getSelectedOption = options => {
  return options.find(item => item.selected === true);
};
```

Reply:
Awesome suggestion! I forgot the  power of `Array.find()`, tks for remember me <3